conans/test/unittests/client/tools/oss/__init__.py

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env python
```

Comment:
shebang python not required.
Also for empty files, the encoding is not required either. Most ``__init__.py`` files should be just empty